xml/admin_saltcluster.xml

```diff
@@ -200,6 +200,17 @@ placement:
     </para>
 <screen>&prompt.cephuser;ceph-salt config /ceph_cluster/roles/bootstrap reset</screen>
    </step>
+   <step>
+    <para>
+      After removing all OSDs on a single host, remove the host from the crush map:
+    </para>
+<screen>&prompt.cephuser;ceph osd crush remove <replaceable>bucket-name</replaceable></screen>
+    <note>
+      <para>
+        The bucket name should be the same as the host name.
```

Comment:
@jecluis if you think we need to be more verbose here, let me know. It's a little vague, but I do feel by this step they should be aware of how to remove this. Might be too much assumed knowledge, however.

Reply:
I think this is clear enough. Unless ones does not read the surrounding text and just goes about running commands blindly, but I think we can hardly blame the docs for that.